### PR TITLE
[Hub Generated] Review request for Microsoft.InstanceMetadataService to add version stable/2021-11-01

### PR DESCRIPTION
**Pull request by ChhabraAnshul**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/ChhabraAnshul/azure-rest-api-specs/release-imds-Microsoft.InstanceMetadataService-2021-11-01...main/).
# Data Plane API - Pull Request

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).

* Link to engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [ ] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
* Design Document:
* Previous Open API Doc:
* Updated paths:

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

**Comment by joshdmcc** [comment]
Add a suppressions line for the new version (where they're all suppressing camelcase)

**Review by joshdmcc — CHANGES_REQUESTED** [review]

**Review by joshdmcc — CHANGES_REQUESTED** [review]

**Review by joshdmcc — APPROVED** [review]

**Comment by ChhabraAnshul** [comment]
@weidongxu-microsoft it has already been in preview for some time and now we are moving it from preview to GA

**Comment by joshdmcc** [comment]
> Please check https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship
> 
> Usually service should not start with a GA api-version?

What do you mean by this? This PR is just to add documentation for one of our new versions that we are moving public from experimental. I don't believe we've had to do this for any previous additions like this.

**Comment by weidongxu-microsoft** [comment]
> What do you mean by this? This PR is just to add documentation for one of our new versions that we are moving public from experimental. I don't believe we've had to do this for any previous additions like this.

No, they don't. If 2017-03-01 is already GA, and it is only adding a few properties, it does not need a review session.

My ask is most based on that there is no preview api-version ever, https://github.com/Azure/azure-rest-api-specs/tree/main/specification/imds/data-plane/Microsoft.InstanceMetadataService
But you and Anshul are saying "moving from preview to GA" or "moving from experimental to public", which I am not sure how I should interpret it, and whether to treat this as your first GA version or not.

If you are doing preview or experimental, you should use a preview api-version like 2021-11-01-preview.

File: specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2021-11-01/imds.json [hunk 1]
```diff
@@ -581,6 +591,10 @@
         "virtualTpmEnabled": {
           "type": "string",
           "description": "Identifies if the virtual Trusted Platform Module (TPM) is enabled on the VM"
+        },
+        "encryptionAtHost": {
+          "type": "string",
+          "description": "Identifies if encryption at host is enabled on the VM"
         }
```
Comment (weidongxu-microsoft) [inline specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2021-11-01/imds.json:598]: Using a string for boolean is confusing, but given that you've already having it in GA and you having same for a couple of other properties, I guess there is nothing I can do here.

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**